docs/source/notes/randomness.rst

```diff
@@ -0,0 +1,41 @@
+
+Reproducibility
+===============
+
+Completely reproducible results are not guaranteed across PyTorch releases,
+individual commits or different platforms. Furthermore, results need to be
+reproducible between CPU and GPU executions, even when using identical seeds.
+
+However, in order to make computations deterministic on your specific problem on
+one specific platform and PyTorch release, there are a couple of steps to take.
+
+There are several pseudorandom number generators involved in various layers of
+the library, which you will need to seed manually to make runs reproducible.
+
+Numpy
+.....
+The numpy rng can be seeded with::
```

Comment:
I wouldn't put the bit about Numpy here; not everyone is going to use Numpy. I'd put it after PyTorch, and preface it with, "if you, or any of the libraries you are using, use numpy..."

Reply:
Good suggestion; I just searched for `numpy` in the repo and deduced from that that numpy is used somewhere within PyTorch.